mlrun/model_monitoring/writer.py

```diff
@@ -230,6 +232,8 @@ def do(self, event: _RawEvent) -> None:
         logger.info("Starting to write event", event=event)
         self._update_tsdb(event)
         self._update_kv_db(event)
+        logger.info("Completed event DB writes")
```

Comment:
Not sure what the expected traffic is, but logging anything on a per request basis is very verbose.

Reply:
I believe it's still ok at this point, maybe we should remove it in 1.8